auth_backends/_version.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.1.1'
```

Comment:
I originally published a version 0.1.0 to PyPI with a README which PyPI's reST parser couldn't render properly. I deleted the package in an attempt to fix it, only to realize that if a file is deleted from PyPI, it cannot be re-uploaded again with the same filename. So, in order to re-upload the source distribution and wheel, I had to bump the version. You can read more about the issue in [this thread](https://mail.python.org/pipermail/distutils-sig/2015-January/025687.html) from the distutils mailing list.

![](http://i.imgur.com/QZcD7ed.png)